### ISSUE TITLE
v1.2.3 hex package does not contain changes made by #55,56

### ISSUE THREAD

**Issue by skirino** [sanmiguel/websocket_client#57]
The newly-released hex package (v1.2.3) gets warned when compiled with Erlang 19.3 as follows (here I'm using Elixir & mix).
```
$ mix deps.get
Running dependency resolution...
Dependency resolution completed:
  websocket_client 1.2.3
* Getting websocket_client (Hex package)
  Checking package (https://repo.hex.pm/tarballs/websocket_client-1.2.3.tar)
  Fetched package

$ mix compile
===> Compiling websocket_client
src/wsc_lib.erl:171: Warning: crypto:rand_bytes/1 is deprecated and will be removed in a future release; use crypto:strong_rand_bytes/1
src/wsc_lib.erl:227: Warning: crypto:rand_bytes/1 is deprecated and will be removed in a future release; use crypto:strong_rand_bytes/1

Compiling 1 file (.ex)
Generated foobar app

$ grep -n 'rand_bytes' ./deps/websocket_client/src/*
./deps/websocket_client/src/wsc_lib.erl:171:    MaskingKeyBin = crypto:rand_bytes(4),
./deps/websocket_client/src/wsc_lib.erl:227:    base64:encode(crypto:rand_bytes(16)).
```

It seems that v1.2.3 was somehow packaged with the source code identical to v1.2.1.
Could you please publish another release to hex.pm so that we can use warning-free version?

**Comment by sanmiguel** [comment]
Oh wow, I wonder how I managed that...?!

OK, I'll get to this shortly. Apologies for the confusion.

**Comment by sanmiguel** [comment]
I've published [1.2.4](https://hex.pm/packages/websocket_client/1.2.4) now, which has the correct code in it. Thanks for pointing that out, and sorry for the confusion.

**Comment by skirino** [comment]
Thanks!
